### ISSUE TITLE
FNX-4451 ⁃ [Bug][a11y] Custom ETP settings height should be increased to 48dp or larger

### ISSUE THREAD

**Issue by sflorean** [mozilla-mobile/fenix#10656]
### Prerequisites
Make sure you have Google Accessibility Scanner installed.

### Steps to reproduce
1. Go to Settings-> Enhanced Tracking protection -> Custom.
1. Scan the page.

### Expected behavior
There are no suggestions to make any changes.

### Actual behavior
For "Cookies", "Tracking content", "Cryptominers" and "Fingerprinters" suggestions are made that the height to be increased from 44 to 48 dp.

### Device information
* Android device: Samsung Galaxy Note 10 (Android 10).
* Fenix version: 76.0.0-beta.1, Nightly 5/14.

![Screenshot](https://user-images.githubusercontent.com/58466444/81954864-0b37b780-9612-11ea-959c-c6b234bcd81c.jpg)UNITO-UNDERSCORE!20200514-183453!UNITO-UNDERSCORE!Accessibility Scanner!

**Comment by ebalazs-sv** [comment]
Verified as fixed on the latest Fenix Nightly Build 21470624, from 5/26, with Nexus 5 (Android 6.0.1), Motorola Moto G6 (Android 8), Pixel 2 (Android 9).

There are no suggestions for "Cookies", "Tracking content", "Cryptominers" and "Fingerprinters".

I will remove the qa:needed label. Thanks!
